tools/harness-thci/OpenThread.py

```diff
@@ -462,9 +464,23 @@ def __startOpenThread(self):
                 else:
                     self.hasActiveDatasetToCommit = False
 
+            # restore whitelist if rejoin after reset
+            if self.isPowerDown and self.isWhiteListEnabled:
+                self.__enableWhiteList()
+                for addr in self.whiteList:
+                    self.addAllowMAC(addr)
```

Comment:
`addAllowMAC` will call `self.whiteList.add(macAddr)` to add all existing entires in whitelist table to whitelist table again. Here, can we replace with `whitelist add` command instead? Same case for blacklist?